doc/api/v0.yaml

```diff
@@ -2259,36 +2272,130 @@ paths:
                                     type: integer
                                     description: upper bound of the available options, maximum being 16 (not included)
                                     minimum: 1
-                                    maximum: 16
-                        required:
-                          - external_id
-                          - options
-                  required:
-                    - vote_start
-                    - vote_end
-                    - committee_end
-                    - payload_type
-                    - proposals
+                                    maximum: 15
+                          tally:
+                            type: object
+                            description: Tally result
+                            nullable: true
+                            properties:
+                              Public:
+                                type: object
+                                description: Public tally result
+                                required: false
+                                properties:
+                                  result:
+                                    type: object
+                                    properties:
+                                      results:
+                                        type: array
+                                        items:
+                                          type: integer
+                                      options:
+                                        type: object
+                                        properties:
+                                          range:
+                                            type: object
+                                            properties:
+                                              start:
+                                                type: integer
+                                                description: starting value of the vote option, starting from 0
+                                                minimum: 0
+                                                maximum: 15
+                                              end:
+                                                type: integer
+                                                description: upper bound of the available options, maximum being 16 (not included)
```

Comment:
I think the maximum is actually 255

Reply:
Are you sure ? An upper bound for the `start` is correct ?

Reply:
You're right, but I swear I remember there was a limit of 256 regarding choices somewhere.
Maybe it's just a vote that can be constructed for up to 256 options, but it's not currently allowed to create a proposal with that many choices